axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/web/BankReconciliationController.java

```diff
@@ -490,4 +490,43 @@ public void computeTotalOfSelectedMoveLines(ActionRequest request, ActionRespons
       TraceBackService.trace(response, e);
     }
   }
+
+  public void correctButtonVisible(ActionRequest request, ActionResponse response) {
+    try {
+      BankReconciliation bankReconciliation = request.getContext().asType(BankReconciliation.class);
+      response.setAttr(
+          "correctBtn",
+          "hidden",
+          Beans.get(BankReconciliationService.class).getIsCorrectButtonHidden(bankReconciliation));
+    } catch (Exception e) {
+      TraceBackService.trace(response, e);
+    }
+  }
+
+  public void correctedLabelFill(ActionRequest request, ActionResponse response) {
+    try {
+      BankReconciliation bankReconciliation = request.getContext().asType(BankReconciliation.class);
+      if (bankReconciliation.getCorrected()) {
+        response.setAttr(
+            "correctedLabel",
+            "title",
+            Beans.get(BankReconciliationService.class)
+                .getCorrectedLabel(bankReconciliation.getCorrectedDate(), request.getUser()));
```

Comment:
Not sure about the way to get the user. The user to take into account is the bankReconciliation.correctedUser not the current user.